model/registry/messaging.yaml

```diff
@@ -207,6 +207,21 @@ groups:
         brief: >
          The ordering key for a given message. If the attribute is not present, the message does not have an ordering key.
         examples: 'ordering_key'
+      - id: gcp_pubsub.message.ack_id
+        type: string
+        brief: >
+         The ack id for a given message.
+        examples: 'ack_id'
```

Comment:
This is a string returned by the [service](https://cloud.google.com/pubsub/docs/reference/rpc/google.pubsub.v1#google.pubsub.v1.ReceivedMessage), I don't think there are any guidelines on what the format would be.

An example I have from a recently published message is `
PkVTRFAGFixdRkhRNxkIaFEOT14jPzUgKEUSCQZneDsKRhkKbTNcdQdRDRh6LjckYwkTVQYWWXdVWzM9cJG-xJpnLzhzfWB9Y1ISAABNV35WWB4OanSMh_vm8MiEXXtJUpfxsJ9zc-jj5vs8Zic9WhJLLD5-My5FQV5AEkw4G0RJUytDCypYEU4EISE-MA`, so I'm not sure if it would help to have that as an example. What do you think?

Reply:
I see, I thought it would a UUID or something, but I agree, doesn't make sense to add that long string :D. All good.